src/style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir" ;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.input {
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+  height: 70px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  padding-left: 62px;
+  color: #3d4e61;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.icon-big {
+  position: absolute;
+  margin: 25px 26px 0;
+}
+
+.input--small {
+  font-size: 14px;
+  line-height: 19px;
+  height: 42px;
+  padding-left: 33px;
+}
+
+.icon-small {
+  position: absolute;
+  margin: 15px 13px 0;
+  width: 11px;
+  height: 11px;
+}
+
+.form--small:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.form:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
```

Comment:
you don't have `border` in `.form`, but add it in `.forn:hover`, that's why form jump, fix it